arviz/plots/ppcplot.py

```diff
@@ -127,7 +127,9 @@ def plot_ppc(
 
         >>> import arviz as az
         >>> data = az.load_arviz_data('radon')
-        >>> az.plot_ppc(data)
+        >>> az.plot_ppc(data,data_pairs={"obs":"obs"})
+        >>> #az.plot_ppc(data,data_pairs={"obs":"obs_hat"})
+            
```

Comment:
I think there is some white space here, so I think you need to remove it.

Reply:
@ahartikainen Do you mean line 132 in my commit? But I feel that space was there in the form of line 131 in the original file

Reply:
You just probably need to remove trailing whitespace

Reply:
@amukh18 It looks like you have some spaces (probably 12) in line 132, you should remove them and leave only the new line